explorer/explorerroutes.go

```diff
@@ -1121,7 +1121,11 @@ func (exp *explorerUI) Search(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	if err != nil {
-		exp.StatusPage(w, "search failed", err.Error()+", you can ", WrongNetworkStatusType)
+		if err.Error() == "Looks like you are searching for an address of type P2PK" {
+			exp.StatusPage(w, "search failed", err.Error(), NotFoundStatusType)
```

Comment:
Instead of doing exp.StatusPage followed by return here and outside the if, just set the status type in a variable first, then change the variable for this condition, then do exp.Status page and return once.